<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QMainWindow" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CodingSpace">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>881</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RLV_Label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>                           RAM Live View</string>
    </property>
   </widget>
   <widget class="QLabel" name="Console_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QLabel" name="Log_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Application Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>410</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Halt_Button">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Halt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>550</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LogTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>811</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ConsoleTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>811</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ref</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="AddressBrowser">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>141</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ValueBrowser">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>60</y>
      <width>101</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NameBrowser">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>60</y>
      <width>101</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ReferenceBrowser">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>60</y>
      <width>61</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LineBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>321</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="CollectorButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Collector</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1339</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuC_IDE">
    <property name="title">
     <string>C! - IDE</string>
    </property>
   </widget>
   <addaction name="menuC_IDE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
